cli/internal/kubernetes/upgrade.go

```diff
@@ -73,10 +79,19 @@ type Upgrader struct {
 	imageFetcher     imageFetcher
 	outWriter        io.Writer
 	log              debugLog
+	tf               terraformUpgrader
+}
+
+// a terraformUpgrader performs the Terraform interactions in an upgrade.
+type terraformUpgrader interface {
```

Comment:
nit: should be moved further down in the file. [ref](https://github.com/edgelesssys/wiki/blob/master/company/gostyle.md#order-of-elements-in-a-source-file).